src/contrib/cluster/Akka.Cluster.Tools/Singleton/ClusterSingletonManager.cs

```diff
@@ -612,8 +638,7 @@ private void GetNextOldestChange()
         {
             var newOldest = handOverTo?.Path.Address;
             Log.Info("Singleton terminated, hand-over done [{0} -> {1}]", _cluster.SelfAddress, newOldest);
-            handOverTo?.Tell(HandOverDone.Instance);
```

Comment:
Merge mistake on my part when I rebased this PR after yours had been merged in. I'll undo it.